node/service.go

```diff
@@ -37,6 +37,10 @@ type ServiceContext struct {
 	AccountManager *accounts.Manager
 }
 
+func NewServiceContext(conf *Config, srv map[reflect.Type]Service, mux *event.TypeMux, am *accounts.Manager) *ServiceContext {
```

Comment:
This function is only used in the test codes. Do we need this function in the main code?

Reply:
1)`ServiceContext` has private member variables needed in the test.
2) There are no other ways to set those variables from other packages.

Reply:
Got it. Thanks!